x-pack/plugin/ml/build.gradle

```diff
@@ -79,6 +79,7 @@ dependencies {
   compileOnly project(path: xpackModule('core'))
   compileOnly project(path: xpackModule('autoscaling'))
   compileOnly project(path: xpackModule('ml-package-loader'))
+  compileOnly project(path: xpackModule('ql'))
```

Comment:
Do we need to depend on the `ql` module? If so, why?

`ql` is a dependency of `esql`, but we're now accessing ES|QL external interface classes via `core`.

If there is currently a genuine need to depend on `ql` here, maybe that's a sign that something else ought to be refactored so we don't.

Reply:
Good catch. This shouldn't be necessary, and is a leftover of the previous attempt.